queries/jsonnet/highlights.scm

```diff
@@ -30,63 +70,64 @@
   ","
   ";"
   ":"
-  "::"
-  ":::"
 ] @punctuation.delimiter
 
-; Nested field inheritance
-("+" @punctuation.delimiter [":" "::" ":::"])
-
-"=" @operator
-(unaryop) @operator
-(binary
-  operator: (_) @operator)
-
-"for" @repeat
-
-"function" @keyword.function
-
-"in" @keyword.operator
-
 [
- (local)
- (tailstrict)
- "assert"
-] @keyword
+  "::"
+  ":::"
+] @punctuation.special
 
+(field
+  (fieldname) "+" @punctuation.special)
+
+; Imports
 [
-  "else"
-  "if"
-  "then"
-] @conditional
+  (import)
+  (importstr)
+] @include
 
-"error" @exception
+; Fields
 
-[
-  (dollar)
-  (self)
-  (super)
-] @variable.builtin
-((id) @variable.builtin
- (#eq? @variable.builtin "std"))
+(fieldname (id) @field)
+(fieldname (string
+             (string_start) @text.strong
+             (string_content) @field
+             (string_end) @text.strong
+           ))
 
-; Function declaration
+; Functions
+(field
+  function: (fieldname (id) @function))
+(field
+  function: (fieldname
+              (string
+                (string_start) @text.strong
+                (string_content) @function
+                (string_end) @text.strong
+              )))
 (param
   identifier: (id) @parameter)
-(field
-  (fieldname (id) @function)
-  [(anonymous_function) (params)])
-(bind
-  function: (id) @function)
+
+(bind (id) @define)
```

Comment:
remove this

Reply:
Considering the `bind` here is a local variable, would `@variable.local` be reasonable?

Reply:
that's fine but this is more a locals thing